Formula/s2n.rb

```diff
@@ -1,8 +1,8 @@
 class S2n < Formula
   desc "Implementation of the TLS/SSL protocols"
-  homepage "https://github.com/aws/s2n-tls"
-  url "https://github.com/aws/s2n-tls/archive/refs/tags/1.0.3.tar.gz"
-  sha256 "7e9783cbaa1b14bebed31f4f85019adb39222e36f59f10da86be5b1f2a3bea85"
+  homepage "https://github.com/awslabs/s2n"
+  url "https://github.com/awslabs/s2n/archive/v1.0.4.tar.gz"
+  sha256 "ca283d856043d2bef892708ea7dcd966d168b051956a4df43c9829f782b82121"
```

Comment:
Source shouldn't change

Reply:
Plus https://github.com/awslabs/s2n redirects to https://github.com/aws/s2n-tls anyway, so may as well skip the redirection.